django/conf/global_settings.py

```diff
@@ -157,6 +157,7 @@ def gettext_noop(s):
 LANGUAGE_COOKIE_AGE = None
 LANGUAGE_COOKIE_DOMAIN = None
 LANGUAGE_COOKIE_PATH = '/'
+LANGUAGE_COOKIE_SAMESITE = 'lax'
```

Comment:
I wouldn't bother, there's nothing bad you can do with cross-origin language headers.